Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.xaml

```diff
@@ -29,77 +30,71 @@
+                                    <ListView x:Name="OptionsListView"
+                                              Grid.Row="2"
+                                              Width="{TemplateBinding OpenPaneLength}"
+                                              IsItemClickEnabled="True"
+                                              Margin="0,20,0,8"
+                                              VerticalAlignment="Bottom"
+                                              AutomationProperties.Name="Option items"
+                                              ItemTemplate="{TemplateBinding OptionsItemTemplate}"
+                                              ItemTemplateSelector="{TemplateBinding ItemTemplateSelector}"
+                                              ItemsSource="{TemplateBinding OptionsItemsSource}"
+                                              ScrollViewer.VerticalScrollBarVisibility="Disabled"
+                                              SelectedIndex="{Binding SelectedOptionsIndex, Mode=TwoWay, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+                                              SelectedItem="{Binding SelectedOptionsItem, Mode=TwoWay, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+                                              SelectionMode="None"
+                                              TabIndex="2">
+                                        <ListView.ItemContainerStyle>
+                                            <Style TargetType="ListViewItem">
+                                                <Setter Property="Padding" Value="0" />
+                                            </Style>
+                                        </ListView.ItemContainerStyle>
+                                    </ListView>
                                 </Grid>
                             </SplitView.Pane>
                             <ContentPresenter x:Name="ContentPart"
                                               AutomationProperties.Name="Content"
                                               Content="{TemplateBinding Content}" />
                         </SplitView>
-                        <Button Name="HamburgerButton"
+                        <Button x:Name="HamburgerButton"
                                 Width="{TemplateBinding HamburgerWidth}"
                                 Height="{TemplateBinding HamburgerHeight}"
                                 Visibility="{TemplateBinding HamburgerVisibility}"
+                                Margin="0,8"
                                 Padding="0"
                                 VerticalAlignment="Top"
                                 AutomationProperties.Name="Main button"
                                 Background="Transparent"
                                 BorderThickness="0"
                                 TabIndex="0">
-                            <ContentControl Margin="{TemplateBinding HamburgerMargin}"
```

Comment:
Was removing the HamburgerMargin intentional? If so should we mark that property as deprecated?